drivers/ethernet/eth_stm32_hal.c

```diff
@@ -441,7 +480,10 @@ static int eth_initialize(struct device *dev)
 	HAL_ETH_DMARxDescListInit(heth, dma_rx_desc_tab,
 		&dma_rx_buffer[0][0], ETH_RXBUFNB);
 
-	HAL_ETH_Start(heth);
+	hal_ret = HAL_ETH_Start(heth);
+	if (hal_ret != HAL_OK) {
+		LOG_ERR("HAL_ETH_Start{_IT} failed");
```

Comment:
After this error function continues, is this wanted?

Reply:
Yes. My implementation just extends the original one by error logging.